_field-types/mapping-parameters/format.md

```diff
@@ -0,0 +1,137 @@
+---
+layout: default
+title: Format
+parent: Mapping parameters
+grand_parent: Mapping and field types
+nav_order: 50
+has_children: false
+has_toc: false
+---
+
+# Format
+
+The `format` mapping parameter specifies the date formats that a date field can accept during indexing. By defining the expected date formats, you ensure that date values are correctly parsed and stored, facilitating accurate search and aggregation operations.
```

Comment:
Could you link to date format documentation? https://opensearch.org/docs/latest/field-types/supported-field-types/date/#formats